examples/displayer.rs

```diff
@@ -0,0 +1,324 @@
+        .await?
+        .into_rows_result()?;
+
+    let mut displayer = result2.rows_displayer();
+    displayer.set_blob_displaying(ByteDisplaying::Ascii);
+    println!("\nblob, double, float, time, timestamp:");
+    println!("{}", displayer);
+
+    // example 3 - date, duration, ip address, timeuuid
+
+    session
+    .query_unpaged(
+        "CREATE TABLE IF NOT EXISTS examples_ks.basic6 (a int, timud timeuuid, date1 date, ipaddr inet, dur duration, primary key (a))",
+        &[],
+    )
+    .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic6 
+        (a, timud, date1, ipaddr, dur) 
+        VALUES 
+        (1, now(), '2021-01-01', '3.14.15.9', 1h);",
+            &[],
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic6 
+        (a, timud, date1, ipaddr, dur) 
+        VALUES
+        (3, NOW(), '2024-01-15', '128.0.0.1', 89h4m48s137ms);", // cqlsh prints this as 89.08003805555556h4.8022833333333335m48.137s137.0ms
+            &[],
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic6 
+        (a, timud, date1, ipaddr, dur) 
+        VALUES
+        (4, NOW(), '2024-01-15', '192.168.0.14', 13y2w89h4m48s137ms);",
+            &[],
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic6 (a, timud, date1, ipaddr, dur) 
+        VALUES (2, NOW(), '2024-02-20', '2001:0db8:0:0::1428:57ab', 5d2h);",
+            &[],
+        )
+        .await?;
+
+    session
+        .query_unpaged(
+            "INSERT INTO examples_ks.basic6 (a, timud, date1, ipaddr, dur) 
+        VALUES (5, NOW(), '-250000-02-20', '2001:db8::1428:57ab', 1y1mo1w1d1h1m1s700ms);",
+            &[],
+        )
+        .await?;
+
+    let result2: QueryRowsResult = session
+        .query_unpaged("SELECT * FROM examples_ks.basic6", &[])
+        .await?
+        .into_rows_result()?;
+
+    let mut displayer = result2.rows_displayer();
+    println!("\ndate, duration, ip address, timeuuid:");
+    println!("{}", displayer);
+
+    displayer.set_terminal_width(80);
+    displayer.use_color(false);
+    println!("\nno color, width = 80:");
+    println!("{}", displayer);
+
+    // example 4 - List
+    // Create a table with a list of text
+    session
+    .query_unpaged(
+        "CREATE TABLE IF NOT EXISTS examples_ks.upcoming_calendar ( year int, month int, events list<text>, PRIMARY KEY ( year, month) )",
+        &[],
+    )
+    .await?;
+
+    // Insert some data
+    session
+    .query_unpaged(
+        "INSERT INTO examples_ks.upcoming_calendar(year, month, events) VALUES (2015, 6, ['e1', 'e2', 'e3'])",
+        &[],
+    )
+    .await?;
+
+    let result4: QueryRowsResult = session
+        .query_unpaged("SELECT * FROM examples_ks.upcoming_calendar", &[])
+        .await?
+        .into_rows_result()?;
```

Comment:
:wrench: :recycle: Numbering of the `result` variables is messy. There are multiple `result4`s. Instead, let's just use `result` for each subexample, and let's put all code related to a single subexample (e.g., a list) in a separate code block (`{}`).